aiohttp/connector.py

```diff
@@ -597,10 +598,18 @@ def __init__(self, *, verify_ssl=True, fingerprint=None,
 
         self._use_dns_cache = use_dns_cache
         self._cached_hosts = _DNSCacheTable(ttl=ttl_dns_cache)
+        self._throttle_dns_events = {}
         self._ssl_context = ssl_context
         self._family = family
         self._local_addr = local_addr
 
+    def close(self):
```

Comment:
@fafhrd91 and @asvetlov now the throttled requests are canceled if the connector is closed explicitly. Not really sure if I'm happy with the implementation, to make it possible I had to save all waiters and chain them with the waiters related to the `Event`, take a look alternatives are welcomed.